userguide/directoryservices.rst

```diff
@@ -133,12 +133,12 @@ advanced options.
    | Encryption Mode          | drop-down     | ✓        | Choices are *Off*, *SSL*, or *TLS*. Refer to `SSL vs. TLS                                                                     |
    |                          | menu          |          | <https://www.globalsign.com/en/blog/ssl-vs-tls-difference/>`__ for more information on SSL and TLS.                           |
    |                          |               |          |                                                                                                                               |
-   |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Certificate              | drop-down menu| ✓        | Select the certificate of the Active Directory server if SSL connections are used. If a certificate does not exist, create    |
-   |                          |               |          | a :ref:`Certificate Authority <CAs>`, then create a certificate on the Active Directory server. Import the certificate to the |
+   | Certificate              | drop-down     | ✓        | Select the certificate of the Active Directory server if SSL connections are used. If a certificate does not exist, create    |
+   |                          | menu          |          | a :ref:`Certificate Authority <CAs>`, then create a certificate on the Active Directory server. Import the certificate to the |
    |                          |               |          | %brand% system using the :ref:`Certificates` menu.                                                                            |
    |                          |               |          |                                                                                                                               |
+   |                          |               |          | Choose the blank entry and click :guilabel:`SAVE` to clear a saved certificate from the AD configuration.                     |
```

Comment:
This reads like it is part of the previous procedure to add a certificate.  Swap the "to clear a saved cert" part to the front to differentiate.

```suggestion
   |                          |               |          | To clear a saved certificate, choose the blank entry and click :guilabel:`SAVE`.                     |
```

(Fix table whitespace, too.)